x/gamm/keeper/swap_test.go

```diff
@@ -184,7 +184,7 @@ func (suite *KeeperTestSuite) TestCalcOutAmtGivenIn() {
 				suite.NoError(err)
 				pool = poolExt.(swaproutertypes.PoolI)
 			} else if test.param.poolType == "concentratedliquidity" {
-				poolExt, err := suite.App.ConcentratedLiquidityKeeper.CreateNewConcentratedLiquidityPool(ctx, 1, "bar", "foo", sdk.MustNewDecFromStr("70.710678118654752440"), sdk.NewInt(85176))
+				poolExt, err := suite.App.ConcentratedLiquidityKeeper.CreateNewConcentratedLiquidityPool(ctx, 1, "bar", "foo", 1)
```

Comment:
Can we use the new `PrepareDefaultPool` helper here?

Reply:
Are you able to import functions from one test package into another?

Reply:
WDYT about re-defining this helper on the `apptesting` suite and naming it `PrepareConcentratedPool`. I think it would also be helpful for swaprouter tests